### PR TITLE
fbzmq: Add fbzmq library

### PR DESCRIPTION
**Pull request by ammubhave**
fbzmq provides a framework for writing services in C++ while leveraging libzmq (message passing semantics).

[Dependency of openr]

Compile tested: nbg6817, master

Maintainer: me

Signed-off-by: Amol Bhave <ambhave@fb.com>

### CODE REVIEW

File: libs/fbzmq/Makefile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+PKG_BUILD_PARALLEL:=1
+CMAKE_SOURCE_SUBDIR:=fbzmq
+CMAKE_OPTIONS:= \
+	-DCMAKE_POSITION_INDEPENDENT_CODE=ON \
```
Comment (diizzyy) [inline libs/fbzmq/Makefile]: Doesn't $(FPIC) work?

Comment (neheb) [reply libs/fbzmq/Makefile]: Given that dependent packages use this, I would not touch it.

File: libs/fbzmq/Makefile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	-DBUILD_TESTS=OFF \
+	-DBUILD_SHARED_LIBS=ON \
+	-DTHRIFT1="$(STAGING_DIR_HOST)/bin/thrift1"
+CMAKE_INSTALL:=1
+
+define Package/fbzmq
+	SECTION:=libs
+	CATEGORY:=Libraries
+	TITLE:=Facebook ZeroMQ wrappers.
+	DEPENDS:=+glog +gflags +libfolly +libzstd +libsodium +libfizz +fbthrift \
```
Comment (neheb) [inline libs/fbzmq/Makefile]: Some of these can be removed as they are implied by the others.

Comment (neheb) [reply libs/fbzmq/Makefile]: Can be reduced to +fbthrift +libzmq +libsigar

**Pull request merged**